easybuild/easyconfigs/c/carma/carma-2.01-foss-2019b.eb

```
@@ -0,0 +1,34 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Helena Gomez <helena.gomez@hpcnow.com>, Erica Bianco, Danilo Gonzalez, Jordi Blasco
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'carma'
+version = '2.01'
+
+homepage = 'http://utopia.duth.gr/~glykos/Carma.html'
+description = """carma - A molecular dynamics analysis program"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['http://utopia.duth.gr/~glykos/progs/']
+sources = ['%(name)s.tar.gz']
+checksums = ['f32cfc2e348691f252d3593e281303d1202e31befba14565aba022331d50b9f3']
+
+preinstall_cmd = 'cd %(builddir)s/%(name)s-%(version)s/src && '
+preinstall_cmd += '$CC $CFLAGS -O -c Color_carma.c && '
+preinstall_cmd += '$FC $FFLAGS $LIBBLAS -O Color_carma.o -o carma &&'
+preinstall_cmd += 'cp -p carma ../bin/'
+
+sanity_check_paths = {
+    'files': ['bin/carma'],
+    'dirs': [''],
+}
+
+tests = ['%(installdir)s/bin/carma']
+
+moduleclass = 'lib'
```

Software to execute molecular dynamics are usually classified in `chem`
```suggestion
moduleclass = 'chem'
```

---

All suggestions applied, thanks @lexming